src/app/about/page.tsx

```diff
@@ -26,11 +27,14 @@ import { focusAreasData } from './data/focusAreasData'
 import { reportsData } from './data/reportsData'
 import { generateStructuredData } from './utils/generateStructuredData'
 
-const { header, seo } = attributes
+const aboutPageData = getPageMarkdownData({
+  slug: 'about',
+  zodParser: GenericPageDataSchema.parse,
+})
 
 export const metadata = createMetadata({
   seo: {
-    ...seo,
+    ...aboutPageData.seo,
```

Comment:
It would be nice to extract it? `const { seo, header } = aboutPageData`

Reply:
Usually I'm all for it. But in this case I think it's best not to, because of conflict variable names.

On the home page, for instance, `homePageData.featuredEcosystemProjects` would conflict with the already declared `featuredEcosystemProjects`.

So, to be consistent, I didn't destructure any objects.

Reply:
I agree with @barbaraperic - Most of the times, destructuring will work. And it reads much better. (If we really wanted, we could change things like `homePageData.featuredEcosystemProjects` to be `featuredEcosystemProjectSlugs` because that is what they actually are.)

Reply:
✅